runtime/kusama/src/lib.rs

```diff
@@ -597,10 +592,8 @@ impl pallet_fast_unstake::Config for Runtime {
 	type Currency = Balances;
 	type BatchSize = frame_support::traits::ConstU32<64>;
 	type Deposit = frame_support::traits::ConstU128<{ CENTS * 100 }>;
-	type ControlOrigin = EitherOfDiverse<
-		EnsureRoot<AccountId>,
-		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>,
-	>;
+	// TODO: Check if this is low-privilege and maybe consider GeneralAdmin instead.
+	type ControlOrigin = EnsureRoot<AccountId>;
```

Comment:
I'd prefer not to merge with an active TODO, but having this be root-limited seems reasonable to me. Fast-unstake is somewhat security-critical to get right and configuration changes should be fairly rare.